### PR TITLE
update release documentation to use twine (because otherwise didnt work for me)

### PR DESCRIPTION
**Pull request by catstavi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -526,11 +526,33 @@ make docs
 ```
 
 ## Releasing
-release to pypi by bumping the version to something reasonable and running
+You will need to have twine installed, and a `.pypirc` file in your home directory.
```
Comment (vickingan) [inline README.md:529]: How do we install twine?

Comment (catstavi) [reply README.md:529]: `pip3 install twine` -- I can update it to say "with pip" ?

Comment (vickingan) [reply README.md:529]: It would be nice to say "with `pip3 install twine`" as an easy installation instruction!

**Review by vickingan — APPROVED** [review]
🎉 Looks great

**Pull request merged**